src/test/java/htsjdk/samtools/util/IntervalTreeTest.java

```diff
@@ -24,13 +24,18 @@
 package htsjdk.samtools.util;
 
 import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.util.Iterator;
 
+import static htsjdk.samtools.util.IntervalTree.Node.HAS_OVERLAPPING_PART;
+
 /**
  * @author alecw@broadinstitute.org
  */
+@Test(singleThreaded=true) // to assure that the common resources aren't clobbered
```

Comment:
I really think it would be easier to just have init() return an interval tree and call it in each method to get the interval tree to use, but I guess this is fine if a bit more esoteric.